soh/soh/SaveManager.cpp

```diff
@@ -25,7 +25,8 @@ SaveManager::SaveManager() {
     AddLoadFunction("base", 1, LoadBaseVersion1);
     AddLoadFunction("base", 2, LoadBaseVersion2);
     AddLoadFunction("base", 3, LoadBaseVersion3);
-    AddSaveFunction("base", 3, SaveBase);
+    AddLoadFunction("base", 4, LoadBaseVersion4);
+    AddSaveFunction("base", 4, SaveBase);
 
```

Comment:
since this has been simplified to the point of "just add `backupFW`" we should be able to do this without a new save version now, just adding `backupFW` to version 3 should be fine since we aren't changing anything/setting anything different on load